react/features/base/react/components/AbstractContainer.js

```diff
@@ -31,7 +31,10 @@ export default class AbstractContainer extends Component<*> {
          * The style (as in stylesheet) to be applied to this
          * {@code AbstractContainer}.
          */
-        style: PropTypes.object,
+        style: PropTypes.oneOfType([
```

Comment:
@virtuacoplenny Since you are here, can you change this to Flow types? There is an unwritten rule (that I just made up) which says that if you touch `PropTypes` code you automatically volunteer to change it to Flow types.

Reply:
I tried and the subsequent flow errors from other consumers of AbstractContainer were too much for me to handle.

Reply:
I'm happy to help, if need be. I love battling with flow :)

Reply:
I did the stupid thing of writing code until it works. Passes flow now.